.github/workflows/sycl-containers.yaml

```diff
@@ -23,6 +23,11 @@ on:
       - 'devops/scripts/install_build_tools.sh'
       - '.github/workflows/sycl-containers.yaml'
 
+concurrency:
+  #  Cancel a currently running workflow from the same PR, branch or tag.
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Comment:
nit: can we use `github.sha` instead of `github.ref`? Thats what we do [here](https://github.com/intel/llvm/blob/sycl/.github/workflows/sycl-post-commit.yml#L26)